### ISSUE TITLE
Support for new Docker env var: RECYCLARR_CREATE_CONFIG

### ISSUE THREAD

**Issue by rcdailey** [tritones/unraid-templates#8]
See details on my corresponding commit: https://github.com/recyclarr/recyclarr/commit/8155cdd385382c15b03e27ea7801c3189a07eaae

# What Changed?

A new environment variable, `RECYCLARR_CREATE_CONFIG`, has been introduced which instructs the container to create a default `recyclarr.yml` on start up if it does not exist.

By default, this environment variable is set to `false` to avoid impacting existing users.

# Motivation

Unraid presents a unique challenge where one or both of the following are true:

- Users struggle to manually create `recyclarr.yml` due to lack of technical knowledge/expertise.
- Invoking the necessary CLI commands is tedious/challenging on this platform. Most examples in the official docs use Docker Compose which isn't helpful here.

# Requested Changes

I propose the following changes for the Unraid template:

- Introduce a GUI toggle for the `RECYCLARR_CREATE_CONFIG` environment variable that is set to `true` by default.
- Optionally give users the ability to change this to `false` if they so wish.
- Restrict the valid values for this environment variable to either `true` or `false` (can be strings). Other values are not supported by the official Recyclarr container.

**Comment by rcdailey** [comment]
Note also that this is not officially released yet (not available on the `latest` tag). It's currently only available on the `edge` tag.

**Comment by gtronset** [comment]
@rcdailey handled in https://github.com/tritones/unraid-templates/pull/9! Tested, confirmed, etc.:

```
Creating default recyclarr.yml file...
[INF] Created configuration at: /config/recyclarr.yml
Starting cron schedule using: @daily
time="2022-12-03T15:05:41-08:00" level=info msg="read crontab: /tmp/crontab"
```

A couple of notes: the default, as discussed, is set to `false` to avoid impacting existing users. Unfortunately, Unraid only provided a relatively simple interface without boolean toggles or radio buttons, which means that strings have to be used to see the variable. That said, I've added a description of the valid values to help ease confusion (and there are clear references elsewhere in the template description to refer to Recyclarr's Wiki).

It takes approximately two hours for templates to propagate to users (the upstream "app store" has a scanner that runs every two hours).
